starbucks_dw/models/reporting/schema.yml

```diff
@@ -0,0 +1,126 @@
+        tests:
+          - not_null
+
+  - name: customer_offer_responses
+    description: Detailed analysis of customer responses to promotional offers including timing and offer characteristics.
+    columns:
+      - name: customer_id
+        description: Unique identifier for each customer.
+        data_type: text
+        tests:
+          - not_null
+      - name: transaction_id
+        description: Unique identifier for each transaction.
+        data_type: text
+        tests:
+          - not_null
+      - name: offer_id
+        description: Unique identifier for each offer.
+        data_type: text
+        tests:
+          - not_null
+      - name: offer_type
+        description: Type of the offer.
+        data_type: text
+      - name: offer_channel
+        description: Channel used to deliver the offer.
+        data_type: text
+      - name: difficulty_rank
+        description: Difficulty ranking of the offer.
+        data_type: integer
+      - name: duration
+        description: Duration of the offer validity.
+        data_type: integer
+      - name: reward
+        description: Reward value associated with the offer.
+        data_type: numeric
+      - name: transaction_status
+        description: Status of the transaction.
+        data_type: text
+      - name: days_since_start
+        description: Days elapsed since offer start.
+        data_type: integer
+      - name: hours_since_start
+        description: Hours elapsed since offer start.
+        data_type: integer
+      - name: response_flag
+        description: Binary flag indicating if the customer responded to the offer.
+        data_type: integer
+        tests:
+          - accepted_values:
+              values: [0, 1]
+      - name: response_timing
+        description: Categorization of response time (Same Day Response, Quick Response, Delayed Response).
+        data_type: text
+      - name: updated_at
+        description: Last update timestamp of the record.
+        data_type: timestamp
+        tests:
+          - not_null
+  - name: offer_performance
+    description: This model calculates the performance metrics for offers, including the total number of offers sent, the number of responded offers, and the response rate.
+    columns:
+      - name: offer_id
+        description: The unique identifier for the offer.
+      - name: offer_type
+        description: The type of the offer (e.g., discount, promotion).
+      - name: offer_channel
+        description: The channel through which the offer was sent (e.g., email, SMS).
+      - name: total_offers_sent
+        description: The total number of offers sent.
+      - name: total_offers_responded
+        description: The total number of offers that received a response.
+      - name: response_rate
+        description: The rate of responses to the offers, calculated as the total number of responses divided by the total number of offers sent.
+custom_tests:
```

Comment:
This is not the way to create custom tests, and for this reason your build failed. You didn't create any `positve_value` test. To create a custom test, please review this documentation: https://docs.getdbt.com/docs/build/data-tests#singular-data-tests